lib/rex/socket/range_walker.rb

```diff
@@ -417,6 +428,17 @@ def expand_nmap(arg)
     return ranges
   end
 
+  protected
+
+  def valid_cidr_chars(arg)
```

Comment:
This looks like it can be `valid_cidr_chars?`, since you return either `true` or `false` and nothing else.

Reply:
Good point, I'll update.